app/assets/javascripts/shared_functions.js

```diff
@@ -8,3 +8,18 @@ function printErrors(error){
     });
   }
 }
+
+function asyncSearchLanguage(query, callback) {
+  // TODO don't hardcode URL
+  const url = `http://localhost:6543/search?q=${query}`
```

Comment:
this hard-coded URL should be from a config constant, and should use the actual production Language Ontology API